packages/circuit-ui/components/Card/Card.stories.tsx

```diff
@@ -108,20 +108,33 @@ export const WithFooter = () => (
     <Card css={cardStyles}>
       <Content />
       <CardFooter>
-        <ButtonGroup>
-          <Button>Cancel</Button>
-          <Button variant="primary">Confirm</Button>
-        </ButtonGroup>
+        <ButtonGroup
+          actions={{
+            primary: {
+              children: 'Cancel',
+            },
+            secondary: {
+              children: 'Confirm',
+            },
+          }}
```

Comment:
You flipped a couple of buttons when migrating the examples (thanks Chromatic! 🙌):

```suggestion
          actions={{
            primary: {
              children: 'Confirm',
            },
            secondary: {
              children: 'Cancel',
            },
          }}
```